vnet/bin/vnflows-monitor

```diff
@@ -394,7 +344,35 @@ def do_diff(previous, current)
 end
 
 $ooo = "abc"
-$opts = VnflowsMonitorOptions.new
+$opts = {}
+$opts[:count] = 1
+$opts[:head] = 0
+$opts[:pretty] = 1
+$opts[:time] = 2
+$opts[:run] = 0
+OptionParser.new do |op|
+  op.on("-c", "--count N", Float, "Number of iterations, 0 = keep going (default 1)") do |c|
+    $opts[:count] = c
+  end
+  op.on("-h", "--head N", Float, "Show top N lines") do |h|
+    $opts[:head] = h
+  end
+  op.on("-d", "--diff", "Turn on diff mode") do |d|
+    $opts[:diff] = true
+  end
+  op.on("-p", "--pretty N", Float,  "Switch pretty mode [0, 1, 99] (default 1)") do |p|
+    $opts[:pretty] = p
+  end
+  op.on("-t", "--time N", Float, "Time in seconds to pause between iterations (default 2)") do |t|
+    $opts[:time] = t
+  end
+  op.on("-r", "--run S", "Pull flows from previous run number xxx, defaults to most recent run") do |r|
+    $opts[:run] = r
+  end
+  op.on("-i", "--iteration x,y,z", Array, "Pull flows from iteration number x. y and z") do |i|
+    $opts[:iteration] = i
```

Comment:
Either "-i x" which implies -i (x-1), x  OR "-i x,y".   If more than two are given, only the last two are used.

The old code used only Integers.  I assume you are testing this with the floats and strings.